SU-SWS/saa_victoria_subtheme
# Adapt 662  regular links underline

# READY FOR REVIEW

# Summary
- Fix link underline for REGULAR link in card paragraph, and line height in card

# Review By (Date)
- 9/2

# Urgency
- medium, being requested now

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Check regular link inside a card's wysiwyg - should be underlined before hover and on hover
4. Check action links and other icon links, should not have underline by default, only on hover.
5. banners link colors shouldn't be affected or changed in any way
6. line height on icon card's paragraph text should be 1.4em
7. Test that regular link in testimonial banner have underline and cta/other icon links do not have the underline until hover.
8. Make sure that the footer doesn't show icons after links in the main 3 column link area and the legal links (red links can still have icons). On this page, the su-link--external class has been added to the footer links, if you test here the icons are hidden for just these 2 areas of links and just for the su-link--external class.
You can see my testing page with this branch here: https://adaptdemo-dev.sites.stanford.edu/
9. Make sure links look correct in icon card, faux card, top banner, red in page banner, testimonial in page banner, all 3 button types, and footer links.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-662 and ADAPT-663
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
